src/views/Settings.vue

```diff
@@ -263,7 +282,8 @@ export default defineComponent({
       appVersion: "",
       locales: process.env.VUE_APP_LOCALES ? JSON.parse(process.env.VUE_APP_LOCALES) : {"en": "English"},
       currentFacilityDetails: {} as any,
-      isStoreFulfilmentTurnedOn: true
+      orderLimitType: 'custom',
+      fulfillmentOrderLimit: 0
```

Comment:
We need to set this with the value that we are considering the default value when the api fails, or the data is not found, instead of setting it as 0 initially and then changing it afterwards.